tests/routes/test_geostore.py

```diff
@@ -91,7 +91,7 @@ async def test_dataset_version_geostore(async_client, batch_client):
     ############################
 
     # This is the hash obtained by POSTing the sample GeoJSON with Postman
-    sample_geojson_hash = "b9faa657-34c9-96d4-fce4-8bb8a1507cb3"
+    sample_geojson_hash = "41b67a74-4ea2-df3f-c3f3-d7131a645f9a"
```

Comment:
It seemed odd to me that the value suddenly changed. I cannot fully explain on why this happend, but it verified the hashes with production hashes and they match. Hashes also matched in my old  PG11 database. It might have been the postgis version used?

For the future we will have to be careful when updating production system to assure these values stay stable.